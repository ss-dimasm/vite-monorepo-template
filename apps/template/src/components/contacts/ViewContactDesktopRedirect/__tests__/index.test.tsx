import { ViewContactDesktopRedirect } from '..'
import { render } from '../../../../scripts/tests'

describe('ViewContactDesktopRedirect', () => {
  it('should match a snapshot', () => {
    expect(render(<ViewContactDesktopRedirect />).asFragment()).toMatchSnapshot()
  })
})
