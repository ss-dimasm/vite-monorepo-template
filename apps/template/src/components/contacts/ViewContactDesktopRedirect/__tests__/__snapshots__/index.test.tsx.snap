// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ViewContactDesktopRedirect > should match a snapshot 1`] = `
<DocumentFragment>
  <div
    class="mocked-styled-93 el-snack-holder"
  />
</DocumentFragment>
`;
